csharp/ql/lib/ext/Microsoft.AspNetCore.Components.model.yml

```diff
@@ -0,0 +1,11 @@
+extensions:
+  - addsTo:
+      pack: codeql/csharp-all
+      extensible: sourceModel
+    data:
+      - ["Microsoft.AspNetCore.Components", "NagivationManager", True, "get_Uri", "", "", "ReturnValue", "remote", "manual"]
```

Comment:
What about `BaseUri` - should that we considered a remote source as well?

Reply:
It's a lesser concern than `Uri` - it's supposed to only be the base path (`https://www.example.com:443/` in `https://www.example.com:443/path/to/my/resource?id=123456789`)

But after reading discussion online, it can sometimes be the same as `Uri` on non-web uses of Blazor. So I think it makes sense to add